src/commands/serve.rs

```diff
@@ -1,4 +1,13 @@
-async fn serve_files(config: &mut fpm::Config, path: &std::path::Path) -> actix_web::HttpResponse {
+pub struct AppState {
+    // Set from command line to provide access, mostly for dev purpose.
+    pub local_identities: Vec<fpm::user_group::UserIdentity>,
+}
```

Comment:
Lets not read this, and use read the cli on demand how we read the `--test` in `is_test()`. Basically I want the local_identities to be used in exactly one place. Right now this looks an official thing.